### PR TITLE
Refactor/add dotenv

### PR DESCRIPTION
**Pull request by 75asa**
***What I did***
- [x] add dotenv
- [x] add config.ts to read dotenv (it's works constant values)
- [x] update README.md
- [x] make it works w/o CLI params

***Note***
- origin issue is #5 
- @mangano-ito I wonder if you could review constant.ts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,29 +5,44 @@ A tool to crawl Slack channels and show usage report of emojis.
...
+$ cd slack-emoji-usage-report
+$ npm install
 ```
 
 ## Prerequisites
 
 You need your own **Slack API Token** to run this tool. Generate it beforehand if you don't have: https://api.slack.com/apps
 
-You need these permissions: `channels:history`, `channels:read`, `emoji:read` and `reactions:read`.
+You need below permissions.
```
Comment (mangano-ito) [inline README.md]:
```suggestion
You need permissions below:
```

I think this is more natural, but just my opinion!

File: src/config.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const config = require("dotenv").config().parsed;
```
Comment (mangano-ito) [inline src/config.ts:1]: It's a good idea to have configuration store! 🎉 
Maybe we can merge each file's `process.env[...]` / `options.*` into this later.

File: src/config.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  export const SOCKET_TOKEN = process.env.SLACK_SOCKET_TOKEN as string;
+  export const CHANNEL_NAME = process.env.SLACK_CHANNEL_NAME as string;
+  export const PORT = process.env.PORT as string;
+}
+
+export namespace SQLite {
+    export const DB_NAME = process.env.DB_NAME as string;
+}
+
+// TODO: I wanna use slash command by Slack #2
```
Comment (mangano-ito) [inline src/config.ts:17]: I want it, too! :+1:

File: README.md [hunk 4]
```diff
@@ -5,29 +5,44 @@ A tool to crawl Slack channels and show usage report of emojis.
...
 
+#### With args target db & channel name
+
+```sh
+$ env SLACK_TOKEN="<slack token>" npm run crawl -- --db="<database name>" --channel="<channel name>"
+```
+
+#### Using dotenv
+
+If using this mode, first you need run commands `$ cp _env .env`.
```
Comment (mangano-ito) [inline README.md]: Nice! It is more useful to use `dotenv` to load configurations from the file! 😄

Comment (mangano-ito) [inline README.md]:
```suggestion
If using this mode, first you need run the command: `$ cp _env .env`.
```

This is a subtle suggestion. Another nitpicking!

**Review by mangano-ito — APPROVED** [review]
Looks good to me! Thank you! 😄

File: src/config.ts (hunk 3 above)
Comment (75asa) [reply src/config.ts:17]: @mangano-ito thx I'll be make it works on #2  👯

File: src/config.ts (hunk 2 above)
Comment (75asa) [reply src/config.ts:1]: @mangano-ito thx exactly I agree that idea , but I wanna narrow down on this PR 🙄  could u make a issue to merge `optins.*` / `process.env[...]` values into config store files ?

Comment (75asa) [reply src/config.ts:1]: @mangano-ito I just made a issue #7 !! Could u check & edit dat issue if it need 2 fix it edit.

**Comment by 75asa** [comment]
@mangano-ito thx ur reviews, this PR is ready to merge 🚀

**Comment by mangano-ito** [comment]
@75asa I'm sorry to have kept you waiting... Now it's merged!

**Pull request merged**